terrastruct/d2
# apply font size to markdown

## Summary

Apply font size to markdown

## Details
- fixes #529 
- adds md_fontsize_10 test
- update markdown rendering with font-size
- update markdown measurement to account for font-size
- for containers that are md text (should this be valid?), don't apply label font-size increase

### before (default font-size 16)

![Screen Shot 2023-04-12 at 8 26 36 PM](https://user-images.githubusercontent.com/85081687/231641327-07af0a67-4955-4b36-8003-25c51580ad71.png)


### after (font-size 10)

![Screen Shot 2023-04-12 at 8 26 45 PM](https://user-images.githubusercontent.com/85081687/231641331-f1f1cf25-554c-4c58-9459-66218135bf7e.png)
